lib/remoteproc/remoteproc_virtio.c

```diff
@@ -222,6 +245,7 @@ rproc_virtio_create_vdev(unsigned int role, unsigned int notifyid,
 	vdev->notifyid = notifyid;
 	vdev->role = role;
 	vdev->reset_cb = rst_cb;
+	vdev->config_len = vdev_rsc->config_len;
```

Comment:
seems no need of  vdev->config_len field as information is in vdev_rsc that is accessible in rproc_virtio_write_config and rproc_virtio_read_config.

Reply:
Yes, fix.

Reply:
Yes, removed from new PR